api/nodes/serializers.py

```diff
@@ -1590,13 +1590,6 @@ def create(self, validated_data):
         if registration_responses:
             self.update_registration_responses(draft, registration_responses)
 
-        if affiliate_user_institutions:
-            if draft.branched_from_type == DraftNode:
-                affiliated_institutions = draft.creator.affiliated_institutions.all()
```

Comment:
The chunk here is the whole point of the PR, we just don't need the separate behavior for `branched_from_type != DraftNode` because we get that for free in the `create_from_node` call.

Could/should just be
```
if affiliate_user_institutions and draft.branched_from_type == DraftNode:
    draft.affiliated_institutions.set(draft.creator.affiliated_institutions.all())
```